CONTRIBUTING.md

```diff
@@ -221,21 +221,39 @@ To update goldens for all tests, run:
 flutter test test/ --update-goldens
 ```
 
-## Manual Testing
+## Manual testing
 
 To explore DevTools with all experimental features enabled:
 
+### Setup and start
+
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-2. Start DevTools:
+2. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet. 
+
+3. Start DevTools:
 ```
-git clone git@github.com:flutter/devtools.git
+git clone https://github.com/flutter/devtools.git
 ./devtools/tool/update_flutter_sdk.sh
 cd devtools/packages/devtools_app
 ../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
 ```
 
-3. Paste URL of your application (for example [Gallery](#connect-to-application)) to the connection textbox.
+4. Paste URL of your application (for example [Gallery](#connect-to-application)) to the connection textbox.
+
+### Refresh and start
+
+1. Run in the `devtools` created in previous section:
+
+```
+git pull origin master
+./tool/update_flutter_sdk.sh
+cd packages/devtools_app
+flutter pub upgrade
+../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
+```
+
+2. Paste URL of your application (for example [Gallery](#connect-to-application)) to the connection textbox.
```

Comment:
The `#connect-to-application` section mentioned here seems to be out of date since it mentions an observatory url.
We will probably want to update it to reflect the VM Service URL
Also the `#developer` section below it makes reference to the observatory URL so we should fix that too

Reply:
Discussed offline: we use the same language as flutter console.